server/database.sql

```diff
@@ -9,11 +9,14 @@ CREATE TABLE public.User_credentials
 CREATE TABLE Users
 (
   user_id serial NOT NULL,
-  firstname varchar(50)  NULL,
-  lastname varchar(50)  NULL,
+  firstname varchar(50) NOT NULL,
+  lastname varchar(50) NOT NULL,
```

Comment:
Note to self: because we are now forcing users to put in a first and last name at Signup